packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -361,12 +362,12 @@ This behavior is slightly different in the highlight selection style, where sing
 </Flex>
 ```
 
-{/*
+// TODO: comment this section out again
```

Comment:
Will re-comment this stuff out when approved, we can expose these docs when we do the full release